SPARCore/GADRemoteModel.h

```diff
@@ -0,0 +1,19 @@
+//
+//  GADRemoteModel.h
+//  Publications
+//
+//  Created by Addison Gould on 3/5/17.
+//  Copyright © 2017 comp. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface GADRemoteModel : NSObject
+
++ (void) fetchModelsWithParams: (NSDictionary * _Nonnull)queryParams
```

Comment:
The only other thing this function needs is whatever information it needs to alter the base URL to get to the right endpoint. An `NSString` maybe? Or maybe you can do something clever with `NSURL` and `NSURLComponents`